test/Microsoft.IdentityModel.Tokens.Tests/Validation/IssuerValidationResultTests.cs

```diff
@@ -42,6 +43,12 @@ await Validators.ValidateIssuerAsync(
                 else
                 {
                     ValidationError validationError = operationResult.Error;
+                    foreach (StackFrame stackFrame in operationResult.Error!.StackFrames)
```

Comment:
nit: can this be moved into a separate class, maybe ValidationUtils to reduce code duplication?